bosh/secrets.example.yml

```diff
@@ -34,12 +33,40 @@ properties:
     component_name: cf-swarm-containers-broker
     docker_url: secret
 
-
 networks:
 - name: default
-  type: dynamic
+  type: manual
   cloud_properties:
     security_groups:
     - sec_group
     subnet: subnet
+  static:
+  - <changme>
+  dns:
+  - <changeme>
+  gateway: <changeme>
+  reserved:
+  - <changeme>
+  range: <changeme>
```

Comment:
Are all of the existing IPs in a certain range? Do we need to check for or exclude other instances from being created in that range? `/me doesn't actually understand networking`